prime-router/src/testIntegration/resources/datatests/translation-test-config.csv

```diff
@@ -14,8 +14,9 @@ HL7_to_FHIR/sample_co_2_20220803-0001.hl7,,HL7_to_FHIR/sample_co_2_20220803-0001
 HL7_to_FHIR_to_HL7/elims_40_4988249_33033.hl7,,HL7_to_FHIR_to_HL7/elims_40_4988249_33033-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
 HL7_to_FHIR_to_HL7/elims_40_4988249_33033-with-truncation.hl7,,HL7_to_FHIR_to_HL7/elims_40_4988249_33033-with-truncation-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
 HL7_to_FHIR_to_HL7/elims_52_24095_04204575_5120.hl7,,HL7_to_FHIR_to_HL7/elims_52_24095_04204575_5120-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
-HL7_to_FHIR_to_HL7/47_04608646_11024_mega_specimen.hl7,,HL7_to_FHIR_to_HL7/47_04608646_11024_mega_specimen-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
-HL7_to_FHIR_to_HL7/47_32361_04608646_11034_mega_case.hl7,,HL7_to_FHIR_to_HL7/47_32361_04608646_11034_mega_case-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
+HL7_to_FHIR_to_HL7/elims_47_04608646_11024_mega_specimen.hl7,,HL7_to_FHIR_to_HL7/elims_47_04608646_11024_mega_specimen-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
+HL7_to_FHIR_to_HL7/elims_47_32361_04608646_11034_mega_case.hl7,,HL7_to_FHIR_to_HL7/elims_47_32361_04608646_11034_mega_case-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
+HL7_to_FHIR_to_HL7/elims_47_04608717_11184_mega_specimen.hl7,,HL7_to_FHIR_to_HL7/elims_47_04608717_11184_mega_specimen-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
```

Comment:
Took me a minute to recognize that we don't have any duplicates here, the names are just similar until pretty far down the string. Completely optional, but how bout something like this? Would require the files to be renamed as well.
```suggestion
HL7_to_FHIR_to_HL7/elims_47_1_04608646_11024_mega_specimen.hl7,,HL7_to_FHIR_to_HL7/elims_47_1_04608646_11024_mega_specimen-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
HL7_to_FHIR_to_HL7/elims_47_2_32361_04608646_11034_mega_case.hl7,,HL7_to_FHIR_to_HL7/elims_47_2_32361_04608646_11034_mega_case-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
HL7_to_FHIR_to_HL7/elims_47_3_04608717_11184_mega_specimen.hl7,,HL7_to_FHIR_to_HL7/elims_47_3_04608717_11184_mega_specimen-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","metadata/fhir_transforms/senders/CDC-ELIMS/elims-sender-transform"
```